src/composables/useApp.ts

```diff
@@ -19,27 +17,14 @@ export function useApp() {
   async function init() {
     const auth = getInstance();
     state.loading = true;
-    await Promise.all([getSpaces(), getStrategies(), getExplore()]);
+    await Promise.all([getStrategies(), getExplore()]);
     auth.getConnector().then(connector => {
       if (connector) login(connector);
     });
     state.init = true;
     state.loading = false;
   }
 
-  async function getSpaces() {
-    let spacesObj: any = await client.getSpaces();
-    spacesObj = Object.fromEntries(
-      Object.entries(spacesObj).map(space => [
-        space[0],
-        formatSpace(space[0], space[1])
```

Comment:
We should still use formatSpace function, this function is needed for change safeSnap plugin id for example.

Reply:
Yeah I see

Reply:
> do you still see a need for these inside formatSpaces?
No we dont need that